src/publish.proj

```diff
@@ -3,13 +3,17 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <Import Project="$(ToolsDir)PublishContent.targets" />
   <Import Project="$(ToolsDir)versioning.targets" />
+  <Import Project="$(PackagesDir)/$(FeedTasksPackage)/$(FeedTasksPackageVersion)/build/$(FeedTasksPackage).targets" />
```

Comment:
It still feels wrong to have to manually construct this path but I don't know of any better way to do it at this time.

Reply:
The only other way to have it auto import I believe is through a PackageReference.

Reply:
This won't work on platforms with case-sensitivity due to NuGet package cache lowercasing, right? Understandable if that's a non-goal at this point, though.

Reply:
We will likely need to deal with that general problem at some point.